pyrasterframes/src/main/python/docs/reference.pymd

```diff
@@ -10,6 +10,15 @@ ReturnDataType function_name(InputDataType argument1, InputDataType argument2)
 
 For the Scala documentation on these functions, see @scaladoc[`RasterFunctions`][RasterFunctions]. The full Scala API documentation can be found [here][scaladoc].
 
+```python imports, echo=False
```

Comment:
not sure we have anything that runs any more? possibly this file can be converted to `md` instead of `pymd`